File: Loop/Armstrong_Number.py
originalNumber=int(input("Enter the number: "))
countDigit=0
newNumber=0
extract=0              
lst=[]  
number=originalNumber               
while number>0:
    extract=number%10
    number=number//10
    lst.append(extract)
    countDigit+=1
print("The total digits in a number is: ", countDigit)
for num in lst:
    newNumber=num**countDigit+newNumber
print(newNumber)
if originalNumber==newNumber:
    print("The number is Armstrong Number!!!")
else:
    print("The number is not Armstrong Number!!")

File: Loop/armstrongnumber.cpp
//Armstrong number checker

#include <iostream>
using namespace std;
int main()
{
    int number,originalNumber,result=0,digit;
    cout<<"Enter a number: ";
    cin>>number;

    originalNumber=number;

    while(number>0){
        digit=number%10;
        result=result+digit*digit*digit;
        number=number/10;
    }

    cout<<" Sum of cubes of digits is: "<<result<<endl;

    if(originalNumber==result){

        cout<<"Number is armstrong number "<<endl;
    }

    else{

        cout<<"Number is not armstrong number "<<endl;
    }
    return 0;
}